db/migrations/20220330111440_modules_table_ver_four_migration.php

```diff
@@ -77,6 +77,29 @@ public function change()
 
             // Drop the old table
             $this->dropTable('module_old');
+
+            // Add more v4 modules
+            $this->execute('
+            INSERT INTO `module` (`moduleId`, `enabled`, `previewEnabled`, `defaultDuration`, `settings`) VALUES
```

Comment:
Some of these modules are not installed by default in v3, please can you check them again?

Reply:
Would it be clearer if we simply did not install them?